src/Controller/StoreApiController.php

```diff
@@ -212,4 +244,88 @@ public function getPaymentStatus(Request $request, SalesChannelContext $context)
             return new JsonResponse(["isFinal" => true]);
         }
     }
+
+    /**
+     * @RouteScope(scopes={"store-api"})
+     * @OA\Post(
+     *      path="/adyen/set-payment",
+     *      summary="set payment for an order",
+     *      operationId="orderSetPayment",
+     *      tags={"Store API", "Account"},
+     *      @OA\RequestBody(
+     *          required=true,
+     *          @OA\JsonContent(
+     *              @OA\Property(property="paymentMethodId", description="The id of new paymentMethod", type="string"),
+     *              @OA\Property(property="orderId", description="The id of the order", type="string")
```

Comment:
```suggestion
     *              @OA\Property(property="paymentMethodId", description="The ID of the new paymentMethod", type="string"),
     *              @OA\Property(property="orderId", description="The ID of the order", type="string")
```

Reply:
I see now that this is copied from SetPaymentOrderRoute.php

Reply:
Yes I will push for changes to be made in the core, so that we don't have to do this workaround